assets/rbac/resizer_role.yaml

```diff
@@ -22,3 +22,9 @@
    - apiGroups: [""]
      resources: ["events"]
      verbs: ["list", "watch", "create", "update", "patch"]
```

Comment:
adding the resources and permissions for provisioner role should suffice. It can be removed from resizer and registrar.

Reply:
Please test once, by removing from resizer and regitrar roles.

Reply:
Please remove it only from resizer role.